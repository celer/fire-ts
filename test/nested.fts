Hello <%=name%>

<%-../test/nested_child.fts%>

<% for(var xi=0; xi<10;xi++){ %>
	<%- ../test/nested_loop.fts (xi) %>
<% } %>
