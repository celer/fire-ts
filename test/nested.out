Hello 4

Goodby 4


xi is 0

xi is 1

xi is 2

xi is 3

xi is 4

xi is 5

xi is 6

xi is 7

xi is 8

xi is 9

